chart/templates/validate-psp-install.yaml

```diff
@@ -0,0 +1,5 @@
+#{{- if .Values.global.cattle.psp.enabled }}
+#{{- if not (.Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy") }}
+#{{- required "The target cluster does not have the PodSecurityPolicy API resource. Please disable PSPs in this chart before proceeding." "" -}}
```

Comment:
This recently changed to recommend the use of `fail` instead of `require`. Would you mind changing this here (new content has been updated in the issue)?